### ISSUE TITLE
TreeListener is not registered with gedmo/doctrine-extensions 3.10

### ISSUE THREAD

**Issue by sasule** [nettrine/extensions#31]
When using this extension together with gedmo/doctrine-extansions v3.10 then there is no TreeListener registered.

Even manual registration of listener fails.
```
$evm = $em->getEventManager();
$treeListener = new \Gedmo\Tree\TreeListener();
$evm->addEventSubscriber($treeListener);
```

This code should add listener to EventManager but calling $evm->getAllListeners() is empty.

Gedmo/doctrine-extensions v3.9 is OK (no change in configs reuired)

**Comment by Pepino483** [comment]
Also problem with Sortable behavior. 

```
This repository can be attached only to ORM sortable listener. 
/vendor/gedmo/doctrine-extensions/src/Sortable/Entity/Repository/SortableRepository.php:56
```

**Comment by tg666** [comment]
Hi, if you need a hotfix for this issue, you can get inspired here 🙂 
https://gist.github.com/tg666/0747d03d4356061585f9f635f7b054d3

**Comment by thorewi** [comment]
Hi, I guess I fixed it here https://github.com/contributte/doctrine-dbal/pull/73.

For now it's in master.

**Comment by tg666** [comment]
> Hi, I guess I fixed it here [contributte/doctrine-dbal#73](https://github.com/contributte/doctrine-dbal/pull/73).
> 
> For now it's in master.

Ah cool, I missed that PR. Thanks :)
